src/Compat.jl

```diff
@@ -1156,6 +1156,17 @@ if VERSION < v"1.7.0-DEV.1106"
         end
     end
 
+    if VERSION >= v"1.2"
+        # Base.scrub_repl_backtrace only exists as of Julia 1.2,
+        # hence the immediately preceding version guard.
+        function Base.display_error(io::IO, stack::ExceptionStack)
+            printstyled(io, "ERROR: "; bold=true, color=Base.error_color())
+            bt = Any[ (x[1], Base.scrub_repl_backtrace(x[2])) for x in stack ]
```

Comment:
Maybe better to just supply this API for consistency, but use a small hack to make it work on 1.0

```suggestion
            bt = Any[ (x[1], (VERSION >= v"1.2" ? Base.scrub_repl_backtrace : identity)(x[2])) for x in stack ]
```

Reply:
Great idea! I will give that a whirl. Thanks for the suggestion and for reviewing Chris! Much appreciated :).

Reply:
Implemented the suggestion and expanded the tests. Thanks again Chris!